lib/actuators/include/Actuator.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+/*
+ * The Actuator class is an abstracted class that encompasses
+ * DC motors, brushless motors, limit servos, and continuous servos.
+ * What "value" represents depends on the actuator type:
+ *
+ *      Actuator Type       Value
+ *      ------------------------------
+ *      DC Motor            Speed
+ *      Brushless Motor     ???
+ *      Limit Servo         Position
+ *      Continuous Servo    Speed
+ */
+
+class Actuator {
+ public:
+  Actuator()                      = default;  // TODO: Don't want default constructor?
+  Actuator(const Actuator& other) = default;
+  virtual ~Actuator()             = default;
+
+  virtual void setValue(float value) {}
+
+  Actuator& operator=(float value);  // undefined atm
```

Comment:
actually i think its st ppl can do things like my_motor = 100; right? if so this is okay but make sure to do = 0 to make it a pure virtual function

Reply:
tbh I'm not sure if the operator= function should be defined for the Actuator class, since this one returns a reference to an Actuator object and the operator='s specific to each derived class I have rn return references to that derived class. Should this always return a reference to the base Actuator class?

Reply:
yah dont do this overload